src/balena.sbvr

```diff
@@ -733,3 +742,6 @@ Rule: It is necessary that each release1 that has a status that is equal to "suc
 Rule: It is necessary that each release that should be running on a device, has a status that is equal to "success" and belongs to an application1 that the device belongs to.
 Rule: It is necessary that each release that should be running on an application, has a status that is equal to "success" and belongs to the application.
 Rule: It is necessary that each release that contains at least 2 images, belongs to an application that has an application type that supports multicontainer.
+
+-- TODO: This rule should be updated to use the device type arch once that is fully modeled.
+Rule: It is necessary that each device that should be managed by a release, should be managed by a release that has a status that is equal to "success" and belongs to an application that is public and is not host and is for a device type that describes the device.
```

Comment:
this has been updated in the closed API to reflect recent `cpu architecture` changes, be sure to backfill that!

Reply:
sorted.